ecs-agent/tmds/handlers/fault/v1/handlers/handlers_test.go

```diff
@@ -581,7 +580,7 @@ func generateStartBlackHolePortFaultTestCases() []networkFaultInjectionTestCase
 			name:                 fmt.Sprintf("%s fail duplicate chain", startNetworkBlackHolePortTestPrefix),
 			expectedStatusCode:   500,
```

Comment:
Shouldn't this be 400 or 409? @mye956 

In this case, although this means the server can't process the request, this failure is due to customer's configuration that they have injected one same fault.

Reply:
Checked offline. If second same BHP fault is injected, we will return 200 + running which is expected.

Reply:
Will be addressed in the following PR -> https://github.com/aws/amazon-ecs-agent/pull/4386

TLDR, this edge case should never happen